### DISCUSSION TITLE
Compiling Issues

### DISCUSSION THREAD

**Discussion by raylaxing** [emukidid/swiss-gc#520] [Q&A] [answered]
Hi!
I am trying to compile the source code however I am running into the same issue:

`~/swiss-gc/cube/swiss/source/video.c:63:11: error: 'SYS_VIDEO_MPAL' undeclared (first use in this function)`
`   63 |      case SYS_VIDEO_MPAL: return &TVMpal480IntDf;`
`      |           ^~~~~~~~~~~~~~`
`~/swiss-gc/cube/swiss/source/video.c:63:11: note: each undeclared identifier is reported only once for each function it appears in
~/swiss-gc/cube/swiss/source/video.c:64:11: error: 'SYS_VIDEO_PAL' undeclared (first use in this function)`
`   64 |      case SYS_VIDEO_PAL:  return &TVEurgb60Hz480IntDf;`
`      |           ^~~~~~~~~~~~~`
Am I missing something? (Same issue on Ubuntu and msys2 on Windows 10)
I see the error is undeclared identifiers, do I need to build a swissSettings file?
Thanks!

**Comment by Extrems** [comment, marked as answer]
You must use our libogc fork. (https://github.com/emukidid/libogc)

**Reply by raylaxing** [reply]
That worked perfectly! Thanks!
